--- /Users/dex/YD 2026/dexapi/sub2api/frontend/src/components/layout/AuthLayout.vue	2026-03-24 10:36:09
+++ /Users/dex/YD 2026/sub2api HUB/sub2api/frontend/src/components/layout/AuthLayout.vue	2026-03-24 10:06:19
@@ -69,9 +69,9 @@
 
 const appStore = useAppStore()
 
-const siteName = computed(() => appStore.siteName || 'Sub2API')
+const siteName = computed(() => appStore.siteName || 'DexAPI')
 const siteLogo = computed(() => sanitizeUrl(appStore.siteLogo || '', { allowRelative: true, allowDataUrl: true }))
-const siteSubtitle = computed(() => appStore.cachedPublicSettings?.site_subtitle || 'Subscription to API Conversion Platform')
+const siteSubtitle = computed(() => appStore.cachedPublicSettings?.site_subtitle || 'AI API Gateway for Team')
 const settingsLoaded = computed(() => appStore.publicSettingsLoaded)
 
 const currentYear = computed(() => new Date().getFullYear())
